cfme/tests/cloud/test_providers.py

```diff
@@ -853,42 +963,11 @@ def test_market_place_images_azure():
     pass
 
 
-@pytest.mark.manual
-@pytest.mark.tier(1)
-def test_upload_azure_image_to_azure():
-    """
-    Upload image copied from RHCF3-11271 to azure using powershell
-    similiar the script in the setup section.
-
-    Polarion:
-        assignee: anikifor
-        casecomponent: Cloud
-        caseimportance: medium
-        initialEstimate: 1/2h
-        setup: # Video Demo Script - Upload VHD to Azure
-               # You have to log in via Login-AzureRmAccount
-               Get-AzureRmSubscription -SubscriptionId "9ee63d8e-aee7-4121-861c-
-               d67a5b8d231e" -TenantId "2e64678d-2fa8-40ed-8922-c9b8e2c3f100" |
-               Select-AzureRmSubscription
-               $ResourceGroupName = "CFMEQE-Main"
-               $BlobLocation = "https://cfmeqestore.blob.core.windows.net/"
-               $BlobContainer = "templates"
-               $BlobName = "tmpl-osDisk.vhd"
-               $BlobDestination = $BlobLocation + $BlobContainer + "/" + $BlobName
-               $LocalFilePath = "C:\tmp\"
-               $LocalFileName = "cfme-azure-5.6.0.13-1.x86_64.vhd"
-               $LocalFile = $LocalFilePath + $LocalFileName
-               Add-AzureRmVhd -ResourceGroupName $ResourceGroupName -Destination
-               $BlobDestination -LocalFilePath $LocalFile -NumberOfUploaderThreads 8
-        startsin: 5.6
-        title: Upload Azure image to Azure
-    """
-    pass
-
-
-@pytest.mark.manual
+@pytest.mark.long_running
 @pytest.mark.tier(1)
-def test_create_azure_vm_from_azure_image():
+@pytest.mark.provider([RHEVMProvider], selector=ONE_PER_TYPE, override=True,
```

Comment:
Q: just suspect why rhv provider?

Reply:
@digitronik because I need to use this provider as it has the necessary template to provision a VM with powershell and powershell scripts.

Reply:
This should be written a bit more generically, as the fact that CFME QE has this template on a RHEV provider is unique to us.

I think there's an assumption here, that's not immediately apparent to me - there is an azure account required to run this.  Is the azure powershell VM already setup with the account credentials? 

I think that the fixtures for this should be structured to look for a specific definition in cfme-qe-yamls of where an azure powershell vm can be run. This could take different forms, but I think it should use the current structure of `provider.data.templates` to list the template name and creds under a specific key (following the pattern of full_template, console_template, small_template, etc).

The fixtures should look to identify any provider that has one of these VM templates defined, and somehow pick one.
The expected setup of the azure VM (credentials, installed packages) should be defined at least in a docblock, even better as a cloud-init file.